src/utils/resolve-to-type-wrapper.util.mts

```diff
@@ -12,15 +12,6 @@ import { isTypeWrapper } from './is-type-wrapper.util.mjs';
  * @param target the class definition of the target of the decorator
  */
 export function resolveToTypeWrapper(typeOrIdentifier: AnyInjectIdentifier): TypeWrapper {
-  /**
-   * ? We want to error out as soon as possible when looking up services to inject, however
-   * ? we cannot determine the type at decorator execution when cyclic dependencies are involved
-   * ? because calling the received `() => MyType` function right away would cause a JS error:
-   * ? "Cannot access 'MyType' before initialization", so we need to execute the function in the handler,
-   * ? when the classes are already created. To overcome this, we use a wrapper:
-   * ?  - the lazyType is executed in the handler so we never have a JS error
-   * ?  - the eagerType is checked when decorator is running and an error is raised if an unknown type is encountered
-   */
```

Comment:
Was this documentation important?  Should it be replaced with something more relevant, which details how type wrappers currently work?